src/sdk/namespace_fs.js

```diff
@@ -2077,13 +2104,13 @@ class NamespaceFS {
         return this.versioning === versioning_status_enum.VER_SUSPENDED;
     }
 
-    _get_version_id_by_stat({ino, mtimeNsBigint}) {
+    static get_version_id_by_stat({ino, mtimeNsBigint}) {
```

Comment:
Do we require to follow this convention here as in most place we still have internal functions starting with`_`